Healthcare-Hospital-Management-System/Healthcare-Hospital-Management-System/Controllers/DoctorsController.cs

```diff
@@ -1,52 +1,59 @@
-using System.Collections.Generic;
-using Microsoft.AspNetCore.Mvc;
 using HealthcareHospitalManagementSystem.Models;
 using HealthcareHospitalManagementSystem.Services;
+using HealthcareHospitalManagementSystem.Infrastructure;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
 
-namespace HealthcareHospitalManagementSystem.Controllers
+[ApiController]
+[Route("api/[controller]")]
+public class DoctorsController : ControllerBase
 {
-    [ApiController]
-    [Route("[controller]")]
-    public class DoctorsController : ControllerBase
+    private readonly IDoctorService _doctorService;
+    private readonly Logger _logger;
```

Comment:
In scope of this practice is the following point:
- Implement DI in 3 ways (constructor, property, method).
In the code you provided I see only DI using constructor but not with property and method.
Example of DI with property we had Logger at the practice, example of DI with method we had FileStream.

Please rework your PR otherwise you cannot receive the maximum for this assignment.